src/audiomixerboard.cpp

```diff
@@ -807,13 +816,32 @@ void CAudioMixerBoard::ChangeFaderOrder ( const bool        bDoSort,
     }
 }
 
+QString CAudioMixerBoard::GetTitle()
+{
+    QString myTitle = "";
+    if ( eRecorderState == RS_RECORDING )
+    {
+        myTitle = "[" + tr ( "RECORDING ACTIVE" )  + "] ";
+    }
+    return myTitle + tr ( "Personal Mix at the Server: " ) + strServerName;
```

Comment:
Maybe (just to shorten it) remove "at the Server"?

Reply:
Good idea. Yes, you can do it if you like.

Reply:
Done.